docs/INTEGRATION_RECOMMENDATIONS.md

```diff
@@ -113,9 +113,9 @@ Here's an example that demonstrates deduping:
 
 ## Additional Examples
 
-The examples below assume that the `similar` profile has been setup in the Searchspring Management Console (SMC), and that a Snap `bundle.js` script exists on the page and has been configured with a `RecommendationInstantiator`.
+The examples below assume the `similar` profile has been setup in the Searchspring Management Console (SMC), and that a Snap `bundle.js` script exists on the page and has been configured with a `RecommendationInstantiator`.
 
-A typical "similar" profile that would display products similar to the product passed in via the `products` global context variable.
+`similar` profiles display products similar to the product passed in via the `products` global context variable.
```

Comment:
I'd prefer this line to start with a capital letter (as it is a new paragraph) - but capitalizing `similar` isn't correct either... 🤔

Reply:
this would be a sentence restructure, but i was thinking about docs being more in the voice speaking to someone _wanting to do something_ versus describing _how things work_

`If you want to display products with strong similarity use ...`
`If you want to display products that compliment the current product the shopper use...`
`If someone just added to their cart, and you want to upsell use...`
`If you'd like to display the popular/trending products for a certain category use...`

However these self-Snap docs for some dev agency probably isn't the place for that. this would be more focused on JS implementation

Reply:
i just reverted it... and noticed some cypress tests fail?

Reply:
i only like flakes in my croissants